0800-appendix-197559-brule.Rmd

```diff
@@ -0,0 +1,84 @@
+---
+output: html_document
+editor_options: 
+  chunk_output_type: console
+  markdown: 
+    wrap: 72
+---
+
+```{r setup-197559, eval = F}
+knitr::opts_chunk$set(echo=FALSE, message=FALSE, warning=FALSE)
+source('scripts/packages.R')
+source('scripts/tables.R')
+
+```
+
+```{r  load-data-197559}
+my_site <- 197559
+
+```
+
+# Brule Creek - 197559 - Appendix {-}
+
+## Site Location {.unnumbered}
+
+PSCIS crossing `r as.character(my_site)` is located on
+Highway 43 approximately 20km north of the town of
+Sparwood. Crossing
+`r as.character(my_site)` was located approximately 0.7km upstream from the confluence with the Elk River and is the
+responsibility of the Ministry of Transportation and Infrastructure.
+
+<br>
+
+```{r tab-wshd-197559, eval = F}
+fpr::fpr_table_wshd_sum(site_id = my_site) %>% 
+  select(-'Elev Min') %>% 
```

Comment:
We have seen this before.  We should be dealing with this immediately after it is called in the tables.R script.  Otherwise it needs to be done in multiple places.  When we have so few sites we can just look at google earth and add the actual `Elev Min` by clicking on the mouth of the stream on google earth and fixing it 

(ex. https://github.com/NewGraphEnvironment/fish_passage_bulkley_2022_reporting/blob/0e016dc5e4a8c705dc033bab3325bedb33553d62/scripts/tables.R#L67 )